src/components/ChatRoomHeader.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
+import { FiChevronLeft } from 'react-icons/fi';
+import { IChatRoomHeaderProps } from './Interface';
+
+const ChatRoomHeader = ({
+  currentUserId,
+  setCurrentUserId,
+}: IChatRoomHeaderProps) => {
+  const handleToggleUser = () => {
+    currentUserId === 'user1'
+      ? setCurrentUserId('user0')
+      : setCurrentUserId('user1');
+  };
+
+  return (
+    <Header>
+      <Button>
+        <FiChevronLeft />
+      </Button>
+
+      <ProfileWrapper onClick={() => handleToggleUser()}>
```

Comment:
```suggestion
      <ProfileWrapper onClick={handleToggleUser}>
```
event handle 함수 등록 코드는 일반적으로 이렇게 작성하는것 같습니다.